packages/grafana-ui/src/components/Monaco/types.ts

```diff
@@ -21,10 +21,8 @@ export interface CodeEditorProps {
 
   /**
    * Callback after the editor has mounted that gives you raw access to monaco
-   *
-   * @alpha -- experimental
```

Comment:
just curious if we have any reasoning why this was experimental and why it no longer is

Reply:
Why was it experimental before? I'm unsure, but I think because it was added just for one component to use it.

Why is it not anymore? I didn't really see the need for it tbh - we rely on it for Azure.

Reply:
I'm comfortable removing `experimental` once we have a solid pattern for how 3rd party plugins can load a custom language (easily).

Reply:
```
const handleEditorWillMount = (monaco) => {
  registerMyPromQLLanguage(monaco);
}

<CodeEditor
  language="promql"
  editorWillMount={handleEditorWillMount}
/>
```